examples/07_customers/05_create_customer/variables.json

```diff
@@ -0,0 +1,10 @@
+{
+    "input": {
+      "acceptsMarketing": true,
+      "email": "example@example.com",
+      "firstName": "John",
+      "lastName": "Smith",
+      "password": "qwerty12345",
+      "phone": "1234567890"
```

Comment:
This phone number,  I tried it and it would result in `"Enter a valid phone number`.

In the old customer create example Scott used an (🇦🇺  number) which will work but I changed the commit to an 🇳🇿 number which also work 😉